OMF-Types.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "id": "FirstDynamicType",
+    "name": "First dynamic type",
+    "classification": "dynamic",
+    "type": "object",
+    "description": "not in use",
```

Comment:
Im not a huge fan of not in use, can we do something better there?

Reply:
Updated